.github/workflows/docker-deploy.yml

```
@@ -0,0 +1,26 @@
+name: Docker Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: SSH to server
+        uses: appleboy/ssh-action@v1
+        with:
+          host: ${{ secrets.VPS_HOST }}
+          username: ${{ secrets.VPS_USER }}
+          password: ${{ secrets.VPS_PASS }}
+          script: |
+            cd livecodes
```

If any command fails, the script will continue. Adding `set -e` at the top of the script to exit on error

```suggestion
            set -e
            cd livecodes
```